lib/reducers/rootReducer.js

```diff
@@ -102,6 +103,8 @@ const appReducer = (state = defaultData, action) => {
     case actionTypes.LOAD_DASHBOARD_INFO:
       const { items } = action.dashboardInfo.data;
       const formattedItems = items.map(item => transformChart(item));
+      loadFontsFromDashboard(action.dashboardInfo);
```

Comment:
This should be not a part of reducer logic (side effect). Please consider move it create dedicate action for loading fonts (`thunk`).